activation/handler.go

```diff
@@ -517,63 +517,54 @@ func (h *Handler) handleAtx(ctx context.Context, expHash types.Hash32, peer p2p.
 		return fmt.Errorf("failed to verify atx signature: %w", errMalformedData)
 	}
 
-	logger := h.log.WithContext(ctx).WithFields(atx.ID())
 	existing, _ := h.cdb.GetAtxHeader(atx.ID())
 	if existing != nil {
-		logger.With().Debug("received known atx")
 		return fmt.Errorf("%w atx %s", errKnownAtx, atx.ID())
 	}
 
-	if atx.NIPost == nil {
-		return fmt.Errorf("nil nipst in gossip for atx %s", atx.ShortString())
+	if err := h.SyntacticallyValidate(ctx, &atx); err != nil {
+		return err
```

Comment:
How about adding little context like for `...ValidateDeps()`?
```suggestion
		return fmt.Errorf("atx %v syntactically invalid: %w", atx.ShortString(), err)
```

Reply:
honestly i cannot think of a good context to add to the name. suggestions?